tests/integration/test_main_routes.py

```
@@ -122,3 +122,45 @@ def test_next_up_endpoint_and_ssr(client, device_config_dev):
+    """When no preview image and no plugin_id, show the generic 'Display a plugin' empty state."""
+    import os
+
+    # Remove any preview images so has_preview=False
+    for path in (
+        device_config_dev.processed_image_file,
+        device_config_dev.current_image_file,
+    ):
+        try:
+            os.remove(path)
+        except FileNotFoundError:
+            pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(FileNotFoundError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(FileNotFoundError): ...`

(SIM105)